en/flying/led_meanings.md

```diff
@@ -0,0 +1,38 @@
+# LED meanings
+
+[Pixhawk flight controllers](../flight_controller/pixhawk_series.md) have an RGB LED to indicate the current status of the FMU.
+
+![LED meanings](../../images/led_meanings.gif)
+
+> **Note:** **GPS unit vs GPS Lock** 
```

Comment:
Not sure what this heading is supposed to convey. Perhaps change to "GPS lock is required to take off**"